packages/lit/package.json

```diff
@@ -150,7 +150,10 @@
       "dependencies": [
         "build:rollup",
         "build:ts",
-        "build:ts:types"
+        "build:ts:types",
+        "../lit-html:build",
+        "../lit-element:build",
+        "../reactive-element:build"
```

Comment:
Doesn't `build:ts:types` & `build:ts` all depend on:

```
        "../lit-element:build:ts:types",
        "../lit-html:build:ts:types",
        "../reactive-element:build:ts:types"
```

which all depend on their local `"build:ts"` command?

Do we need this command duplication?

Reply:
`:build` includes the production/rollup outputs, so that you can actually run the thing. `:ts` is `development/`. I've started using `build` to mean "everything you need to run this package" (noted in PR description).

Reply:
That is very clear. I misunderstood the dependencies and accidentally thought it was duplicating transients.